__std__/nix/toolchain/library/r-overlay.nix

```diff
@@ -2,26 +2,7 @@
 
 self: super: {
 
-  rPackages = super.rPackages.override {
-    overrides = ({
-      hexbin = super.rPackages.hexbin.overrideDerivation (attrs: {
-        nativeBuildInputs = attrs.nativeBuildInputs ++ [ super.libiconv ];
-        buildInputs = attrs.buildInputs ++ [ super.libiconv ];
-      });
-    });
-  };
-
-  R = super.R.overrideAttrs (oldAttrs: {
-    # TODO(std) see if this is still needed
-    # Backport https://github.com/NixOS/nixpkgs/pull/99570
-    prePatch = super.lib.optionalString super.stdenv.isDarwin ''
-      substituteInPlace configure --replace \
-        "-install_name libRblas.dylib" "-install_name $out/lib/R/lib/libRblas.dylib"
-      substituteInPlace configure --replace \
-        "-install_name libRlapack.dylib" "-install_name $out/lib/R/lib/libRlapack.dylib"
-      substituteInPlace configure --replace \
-        "-install_name libR.dylib" "-install_name $out/lib/R/lib/libR.dylib"
-    '';
-  });
+  # Use old R 4.1.3 due to https://github.com/tweag/HaskellR/issues/374
+  inherit (inputs.nixpkgs-r.legacyPackages) rPackages R;
```

Comment:
This seems unlikely to work in general: our nixpkgs and this one could diverge on common libraries like `glibc`. I think we should probably instead override the R package to pin the version to an old one.